config/raven.rb

```diff
@@ -0,0 +1,3 @@
+Raven.configure do |config|
```

Comment:
Could you please write a brief description in the commit notes about what `raven` is, and where we can find documentation about it?